README.md

```diff
@@ -28,7 +28,7 @@ This utility provides a quick way to test evaluation of expressions which can be
 % $GOPATH/bin/exptester '@(TITLE("foo"))'
 ```
 
-## Running Tests
+## Development
 
 You can run all the tests (excluding tests in vendor packages) with:
 
```

Comment:
While you are at it you can just change the below to `go test github.com/nyaruka/goflow/...`